Documentation/user-guides/getting-started.md

```diff
@@ -11,63 +11,35 @@ draft: false
+the desired state of a Prometheus and Alertmanager cluster as well as the
+Prometheus configuration.
 
-{{< alert icon="👉" text="Prometheus Operator requires use of Kubernetes v1.16.x and up."/>}}
+The `Prometheus` resource declaratively describes the desired state of a
+Prometheus deployment, while `ServiceMonitor` and `PodMonitor` resources
+describe the targets to be monitored by Prometheus.
 
-> Note: [Operators](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)
-> were introduced by CoreOS as a class of software that operates other software,
-> putting operational knowledge collected by humans into software.
+This guide explains how to use `PodMonitor` and `ServiceMonitor` objects to monitor targets for a sample application.
 
 ## Pre-requisites
 
-To follow this guide, you will need a Kubernetes cluster with admin permissions.
-
-## Installing the operator
-
-The first step is to install the operator's Custom Resource Definitions (CRDs) as well
-as the operator itself with the required RBAC resources.
-
-Run the following commands to install the CRDs and deploy the operator in the `default` namespace:
-
-```bash
-LATEST=$(curl -s https://api.github.com/repos/prometheus-operator/prometheus-operator/releases/latest | jq -cr .tag_name)
-curl -sL https://github.com/prometheus-operator/prometheus-operator/releases/download/${LATEST}/bundle.yaml | kubectl create -f -
-```
-
-It can take a few minutes for the operator to be up and running. You can check for completion with the following command:
-
-```bash
-kubectl wait --for=condition=Ready pods -l  app.kubernetes.io/name=prometheus-operator -n default
-```
-
-The Prometheus Operator introduces custom resources in Kubernetes to declare
-the desired state of a Prometheus and Alertmanager cluster as well as the
-Prometheus configuration. For this guide, the resources of interest are:
+You will need a Kubernetes cluster with admin permissions to follow this guide.
 
-* `Prometheus`
-* `ServiceMonitor`
-* `PodMonitor`
+To install Prometheus Operator, please refer to the instructions [here]({{<ref "installation.md">}}).
 
-The `Prometheus` resource declaratively describes the desired state of a
-Prometheus deployment, while `ServiceMonitor` and `PodMonitor` resources
-describe the targets to be monitored by Prometheus.
+Also, refer to the Platform Guide to learn how to deploy a Prometheus instance before moving ahead.
 
 <!-- do not change this link without verifying that the image will display correctly on https://prometheus-operator.dev -->
 
-![Prometheus Operator Architecture](/img/architecture.png)
+![Prometheus Operator Architecture](../img/serviceMonitor-and-podMonitor.png)
 
 > Note: Check the [Alerting guide]({{< ref "alerting" >}}) for more information about the `Alertmanager` resource.
 
 > Note: Check the [Design page]({{< ref "design" >}}) for an overview of all resources introduced by the Prometheus Operator.
```

Comment:
(suggestion) have a "What's next" section at the end of the doc (like in the [Kubernetes docs](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#what-s-next))?

Reply:
No, I'm more thinking about linking to related pages (e.g. "now that you've completed X, maybe you want to carry on with Y and Z").

Reply:
I feel like the pages should be ordered in a way that people can just click "next" and it makes sense 🤔. What do you think about moving pages around?

Reply:
It's not sufficient IMHO since you may want to suggest multiple pages. See the link I've shared above for the Kubernetes doc.

Reply:
I like the idea of the "what's next" section but then it will eliminate the need for the "Next" button. This is a non-linear approach and we would need to be very careful with all the cross-references and pre-requisites on each page.

A Guide is a step-by-step method to teach users how to accomplish a task. With this, we cannot ensure what direction a user should follow to completely understand the guide.

Reply:
Ack, I agree that it needs a wider discussion. At the same time (and to keep this guide straight to the point), I would adivse to remove links to the Alerting and Design pages here (e.g. it breaks the flow).